modules/rosapps/applications/screensavers/butterflies/lang/ro-RO.rc

```diff
@@ -12,33 +12,33 @@ LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 IDD_DLG_SCREEN DIALOGEX DISCARDABLE  0, 0, 172, 57
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Oprțiuni animație de inactivitate Fluturi"
+CAPTION "Setări pentru economizorul de ecran Fluturi"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON   "Con&firmă",IDOK,7,35,50,14
-    PUSHBUTTON      "A&nulează",IDCANCEL,61,35,50,14
-    CONTROL         "&Rotește Fluturi",ROTATE,"Button",BS_AUTOCHECKBOX |
+    DEFPUSHBUTTON   "OK",IDOK,7,35,50,14
+    PUSHBUTTON      "Revocare",IDCANCEL,61,35,50,14
+    CONTROL         "Rotire a Fluturilor",ROTATE,"Button",BS_AUTOCHECKBOX |
```

Comment:
You stripped the accelerator here. Prove with a pic that romanian 2k3sp2 does not have an accel here. Contrary, if it has, then add one back.

Reply:
I can't prove it as in 2k3sp2 the file is not translated. I mistakenly stripped the accelerator. I will add it back.

Reply:
Fixed!

Reply:
Resolved.